src/format.h

```diff
@@ -164,6 +170,13 @@ class Format {
            type_ == FormatType::kS8_UINT;
   }
 
+  /// Returns true if the format components are scaled
```

Comment:
Is there somewhere I can look up the definition of scaled?  My intuition tells me this is used backwards.  The value "1.0" would be the scaled on, but I don't know graphics jargon all that well.

Reply:
@jaebaek added this code originally ....

Reply:
```
    float scale_factor_for_error_report = 1.0f;	
    if (component->mode == FormatMode::kUNorm ||	
        component->mode == FormatMode::kSNorm ||	
        component->mode == FormatMode::kSRGB) {	
      scale_factor_for_error_report = 255.0f;	
    }
```

is where this came from.

Reply:
Okay, I think the name is correct.  The output want to construct a string that displays values that are not normalized.  If it is "scaled", in the range 0.0-1.0, then it needs to be multiplied by 255, otherwise, it remains the same.

sRGB stands for "standard RGB".

I this "normalized" would be a better term because UNorm and SNorm stand for unsigned and signed normalized values.

Reply:
done.